pkg/graph/resolvers/user_account.go

```diff
@@ -7,12 +7,13 @@ import (
 
 	"github.com/cms-enterprise/easi-app/pkg/authentication"
 	"github.com/cms-enterprise/easi-app/pkg/dataloaders"
+	"github.com/cms-enterprise/easi-app/pkg/sqlutils"
 	"github.com/cms-enterprise/easi-app/pkg/storage"
 )
 
 // UserAccountGetByUsername returns a user account by it's userName
-func UserAccountGetByUsername(store *storage.Store, userName string) (*authentication.UserAccount, error) {
-	return store.UserAccountGetByUsername(userName)
+func UserAccountGetByUsername(ctx context.Context, store *storage.Store, np sqlutils.NamedPreparer, userName string) (*authentication.UserAccount, error) {
```

Comment:
If adding a np here, it could be worthwhile to consider removing the store, and refactoring the store method.
EG
```
func UserAccountGetByUsername(np sqlutils.NamedPreparer, username string) (*authentication.UserAccount, error) 
```